nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/processors/script/ExecuteScript.java

```diff
@@ -91,13 +112,36 @@
 @SeeAlso({InvokeScriptedProcessor.class})
 public class ExecuteScript extends AbstractSessionFactoryProcessor implements Searchable {
 
-    // Constants maintained for backwards compatibility
     public static final Relationship REL_SUCCESS = ScriptingComponentUtils.REL_SUCCESS;
     public static final Relationship REL_FAILURE = ScriptingComponentUtils.REL_FAILURE;
 
     private String scriptToRun = null;
     volatile ScriptingComponentHelper scriptingComponentHelper = new ScriptingComponentHelper();
 
+    /** Whether to use dynamic relationships or not. */
+    private volatile boolean useDynamicRelationships = Boolean.parseBoolean(USE_DYNAMIC_RELATIONSHIPS.getDefaultValue());
+
+    /** Map to keep dynamic property keys and values.
+     * They need to be stored for the case that the value of {@code useDynamicProperties} changes.
+     */
+    private final Map<String, String> dynamicProperties = new ConcurrentHashMap<>();
+
+    private final Set<Relationship> relationships;
+    private ComponentLog log;
+
+    public ExecuteScript() {
+        super();
+        relationships = new ConcurrentSkipListSet<>();
+        relationships.add(ExecuteScript.REL_SUCCESS);
+        relationships.add(ExecuteScript.REL_FAILURE);
+    }
+
+    @Override
+    protected void init(ProcessorInitializationContext context) {
+        super.init(context);
+        log = getLogger();
+        log.warn("The default for USE_DYNAMIC_RELATIONSHIPS will change from \"false\" to \"true\" starting in NIFI 2.0");
```

Comment:
Let's not include this warning, in case we don't make the change in NiFi 2.0 we won't be providing false information

Reply:
done